packages/datatrak-web/src/components/TopProgressBar.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Button } from '@material-ui/core';
+import ArrowBackIosRoundedIcon from '@material-ui/icons/ArrowBackIosRounded';
+
+const Wrapper = styled.div`
+  background: rgba(0, 65, 103, 0.3);
+  width: 100%;
+  min-width: 22.8125rem;
+`;
+
+const ProgressBar = styled.div`
+  background: ${({ theme }) => theme.progressBar.main};
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  height: 0.75rem;
+  width: 100%;
+`;
+
+const ProgressButton = styled(Button)`
+  background: ${({ theme }) => theme.palette.primary.main};
+  color: white;
+  border-radius: 3px;
+  text-transform: none;
+  font-size: 0.9375rem;
+  height: 2.7851875rem;
+  padding: 0.8125rem 1.5625rem;
+`;
+
+const CancelButton = styled(Button)`
+  background-color: white;
```

Comment:
Same with the colours here - this can be set by giving the button a prop of `variant='outlined'` for the outlined button